### DISCUSSION TITLE
Terminal shows successful commit & push but github repo is blank?

### DISCUSSION THREAD

**Discussion by juniperbuilds** [community/community#140731] [New to GitHub]
I cannot for the life of me understand why my repo is empty. I have been trying to trouble shoot for days. I've extensively asked claude for help and between the two of us, there is no understanding as to why my repo remains empty. Please someone help. Thank you

**Comment by PauloDrefahl** [comment]
That's weird, have you tried cloning your repo again to a new environment? Make sure you are also properly logged to GitHub in your environment.  I would also check the branch you are committing to. You can check which branch you're on with: "git branch" If you're not on the right branch, switch to the correct one with "git checkout <branch-name>" Then push again.

Sometimes, even though you pushed, the files might not have been committed properly. Try running: "git status"
If it shows untracked or staged files, you’ll need to commit them first:

git add .
git commit -m "Add files"
git push origin <branch-name>

Lastly, I would check the remote setup. To check run: " git remote -v "
Just to be sure the repo URL matches the one on GitHub. If it doesn't, you might need to set the right one.

**Reply by Keko787** [reply]
It worked for me, thx.

**Reply by juniperbuilds** [reply]
It turns out in an my git history i had some files that were too large to upload. They were no longer in my project however i used AI to help me rewrite the history. Cursor terminal wasn't giving me the errors necessary for me to detect this so i used my device terminal and then finally caught the error. Thanks for the help!

**Reply by ghostinhershell** [reply]
Hi @juniperbuilds 👋🏾,It looks like @PauloDrefahl 's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
